src/v/pandaproxy/test/kafka_client_cache.cc

```diff
@@ -29,19 +29,14 @@ using namespace std::chrono_literals;
 
 namespace pandaproxy {
 struct test_client_cache : public kafka_client_cache {
-    ss::timer<ss::lowres_clock> clean_and_evict_timer;
     ss::gate g;
```

Comment:
I don't recall the outcome of the discussion as to why this gate doesn't appear to be waited on.

Reply:
I call gate close later in the test, that's what I thought was meant by "waiting on a gate"